webrtc.html

```diff
@@ -5059,9 +5070,6 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               as <code>recvonly</code> or <code>inactive</code>,
               as defined in <span data-jsep="subsequent-offers">
               [[!JSEP]]</span>. <!-- onended --></p>
-              <p>When the other peer stops sending a track in this manner, an
-              <code title="event-MediaStreamTrack-ended">ended</code> event is
-              fired at the <code><a>MediaStreamTrack</a></code> object.</p>
```

Comment:
I removed it as it seemed redundant, and I struggled with making it precise.

I guess it could be: *"When the other peer stops sending a track in this manner, the track is removed from any remote streams that were initially sent in the `track` event, and a *muted* event is fired at the track."* ?